app/services/model/AutoCompleteItem.scala

```diff
@@ -16,23 +16,33 @@
 
 package services.model
 
-import services.{DocumentType, HolderOfAuthorisationCode, NationalAdditionalCode, PackageType}
+import services.{DocumentType, HolderOfAuthorisationCode, NationalAdditionalCode}
 
 case class AutoCompleteItem(label: String, value: String)
 
 object AutoCompleteItem {
-  def from(countries: List[Country], value: Country => String = _.countryName): List[AutoCompleteItem] =
+
+  def fromCountry(countries: List[Country], value: Country => String = _.countryName): List[AutoCompleteItem] = {
     countries map (c => AutoCompleteItem(s"${c.countryName} - ${c.countryCode}", value(c)))
+  }
 
-  def from(packageTypes: List[PackageType]): List[AutoCompleteItem] =
+  def fromPackageType(packageTypes: List[PackageType]): List[AutoCompleteItem] = {
     packageTypes map (c => AutoCompleteItem(s"${c.description} - ${c.code}", c.code))
+  }
 
-  def fromDocumentType(documents: List[DocumentType]): List[AutoCompleteItem] =
+  def fromDocumentType(documents: List[DocumentType]): List[AutoCompleteItem] = {
     documents map (d => AutoCompleteItem(s"${d.description} - ${d.code}", d.code))
+  }
 
-  def fromNationalAdditionalCode(codes: List[NationalAdditionalCode]): List[AutoCompleteItem] =
+  def fromNationalAdditionalCode(codes: List[NationalAdditionalCode]): List[AutoCompleteItem] = {
     codes map (c => AutoCompleteItem(c.value, c.value))
+  }
```

Comment:
I'll still vote for removing this. It's not for consistency. All methods before didn't have bracekts, so I don't see any reason to stay with brackets if everyone agree that those methods should live without it.

Some Intellij configuration will pointed those lines as warnings (IntelliJ not sbt) and there will be proposal to remove brackets. I don't block this because of the brackets, but I'm sure that this will be changed by someone in the nearest future to the old implementation.

We're trying to remove brackets from tests, to make it more readable, shorter etc and adding in places that not need it, I don't see any pros of this.

PR looks good, so from me LGTM but without approving